src/main/java/com/fakng/fakngagrgtr/parser/google/GoogleParser.java

```diff
@@ -92,14 +96,7 @@ private LocalDateTime parseLocalDateTime(String datetime) {
     }
 
     private ResponseDTO getPage(int page) {
-        ResponseSpec response = sendRequest(String.format(url, page));
+        ResponseSpec response = getRequest(String.format(url, page));
```

Comment:
Do not make changes at [amazon-parser](https://github.com/fakng-agrgtr/fakng-agrgtr/tree/amazon-parser) branch in separate parser (GoogleParser).
If GoogleParser needs fix create new issue and branch for the issue make fix then PR.
GoogleParser should be reverted

Reply:
These are changes to the base class. it was necessary to move some of the functionality of the Google parser to its base class in order to use them in the Amazon parser. I also changed the name of one method, there is no need to create a separate branch.

Reply:
I agree. Some bureaucracy with branch creation. We're building this quick and stupid sometimes right now
If you have a good motivation for changing something - change it. Additional PRs and branches will slow down the process significantly

Reply:
> These are changes to the base class. it was necessary to move some of the functionality of the Google parser to its base class in order to use them in the Amazon parser. I also changed the name of one method, there is no need to create a separate branch.

Please provide expanded commit history or minimal description to avoid misunderstandings

Sorry guys for complicating you with my bureaucracy 😄 
I've got feedback 💖